File: WebScrapper.AngularJs/WebScrapper.AngularJs/app/models/InformationResult.ts
module app.models{
    export interface IInformationResult {
        id : number,
        cssSelector : string;
        resultAction : IResultAction;
        resultName : string; 
    }

    export class InformationResult implements IInformationResult{        
        constructor(public id : number,
                    public cssSelector : string,
                    public resultAction : IResultAction,
                    public resultName : string ) {                    
        }
    }
}

File: WebScrapper.Api/WebScrapper.core/Layer/Domain/ResponseActions/InformationResult.cs
using System;
using WebScrapper.Core.Helpers.Application;
using WebScrapper.Core.Helpers.Domain;

namespace WebScrapper.Core.Layer.Domain.ResponseActions
{
    public class InformationResult : IDomainEntity
    {
        public Guid Id { get; set; }
        public String CssSelector { get; set; }

        public ResultAction ResultAction { get; set; }

        public String ResultName { get; set; }
    }
}